cpp/source/energy_storage_device.cc

```diff
@@ -2,12 +2,13 @@
  *
  * This file is subject to the Modified BSD License and may not be distributed
  * without copyright and license information. Please refer to the file LICENSE
- * for the text and further information on this license. 
+ * for the text and further information on this license.
  */
 
 #include <cap/resistor_capacitor.h>
 #ifdef WITH_DEAL_II
 #include <cap/supercapacitor.h>
+#include <cap/deal.II/new_supercapacitor.h>
```

Comment:
Well this is the deprecated builder function. I'm curious to see where it is still used.